packages/formation/js/accordion.js

```diff
@@ -160,10 +160,27 @@ const addAccordionClickHandler = () => {
   }
 };
 
+function autoExpandAccordionPanelByUrlHash() {
+  const hash = document.location.hash;
+
+  if (!hash) {
+    return;
+  }
+
+  const anchorId = document.location.hash.slice(1);
+  const accordionButtonSelector = `.usa-accordion li[id="${anchorId}"] .usa-accordion-button`;
```

Comment:
It's necessary to use the verbose `li[id="${anchorId}"]` if an ID leads with digits. That threw me off a bit, https://stackoverflow.com/questions/20306204/using-queryselector-with-ids-that-are-numbers.

The accordion panel's in our site use the Drupal "entityId" as their element ID to ensure uniqueness.